source/install_scripted.rst

```diff
@@ -127,6 +127,12 @@ Options:
     - Contains the fully qualified path to the installation directory for the Quipucords server. Defaults to ``~/quipucords/``.
  - **server_port**
     - Contains the port number for the Quipucords server. Defaults to ``9443``.
+ - **server_username**
```

Comment:
add tagging around the values here, so
``admin``
``qpcpassw0rd``
``admin@example.com``

Reply:
👍